troposphere/static/js/utilities/subscribe.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import AllocationStore from "stores/AllocationStore";
+import BadgeStore from "stores/BadgeStore";
+import ClientCredentialStore from "stores/ClientCredentialStore";
+import ExternalLinkStore from "stores/ExternalLinkStore";
+import HelpLinkStore from "stores/HelpLinkStore";
+import ImageStore from "stores/ImageStore";
+import ImageVersionStore from "stores/ImageVersionStore";
+import ImageVersionMembershipStore from "stores/ImageVersionMembershipStore";
+import ImageVersionLicenseStore from "stores/ImageVersionLicenseStore";
+import ImageVersionScriptStore from "stores/ImageVersionScriptStore";
+import IdentityStore from "stores/IdentityStore";
+import ImageBookmarkStore from "stores/ImageBookmarkStore";
+import InstanceHistoryStore from "stores/InstanceHistoryStore";
+import ImageRequestStore from "stores/ImageRequestStore";
+import InstanceStore from "stores/InstanceStore";
+import InstanceTagStore from "stores/InstanceTagStore";
+import LicenseStore from "stores/LicenseStore";
+import ScriptStore from "stores/ScriptStore";
+import MaintenanceMessageStore from "stores/MaintenanceMessageStore";
+import MyBadgeStore from "stores/MyBadgeStore";
+import MembershipStore from "stores/MembershipStore";
+import ProfileStore from "stores/ProfileStore";
+import ProjectStore from "stores/ProjectStore";
+import ProjectExternalLinkStore from "stores/ProjectExternalLinkStore";
+import ProjectImageStore from "stores/ProjectImageStore";
+import ProjectInstanceStore from "stores/ProjectInstanceStore";
+import ProjectVolumeStore from "stores/ProjectVolumeStore";
+import ProviderMachineStore from "stores/ProviderMachineStore";
+import ProviderStore from "stores/ProviderStore";
+import ResourceRequestStore from "stores/ResourceRequestStore";
+import IdentityMembershipStore from "stores/IdentityMembershipStore";
+import StatusStore from "stores/StatusStore";
+import SSHKeyStore from "stores/SSHKeyStore";
+import QuotaStore from "stores/QuotaStore";
+import SizeStore from "stores/SizeStore";
+import TagStore from "stores/TagStore";
+import UserStore from "stores/UserStore";
+import VersionStore from "stores/VersionStore";
+import VolumeStore from "stores/VolumeStore";
+import AllocationSourceStore from "stores/AllocationSourceStore";
+
```

Comment:
I was thinking 
```
import store1 from 'stores';
import subscribe from 'subscribe';
const myComponent = react.createClass({ ... });
export default subscribe(component, [store1]);
```
```
export default subscribe = (component, storeNames) => {
     storeNames.forEach( name => {
          subscriptions[name] = stores[name];
     });
     ...
}
```
I don't know what you mean by "dynamic library set up" so that might be what I'm not getting. If what you're doing works I can get behind it. Much better than what we are doing now and we can make better in another pass.